### DISCUSSION TITLE
Consider fork Contributors, at least for Top Contributors widget

### DISCUSSION THREAD

**Discussion by olegcherev** [devactivity-app/devActivity-Community#27] [Ideas & Feature Requests]
Some of the public projects have a number of popular forks, syncing code from time to time. It might be a good idea to include Contributors from forks

**Comment by olegcherev** [comment]
> If you add option to enable auto-add of authors of Issues closed as Completed or tagged w/ label indicative of an app contribution like > 'Enhancement' or 'Bug', or allow me to enter custom contributors, then I would use your software right away

**Reply by damonexsppfx-alt** [reply]
How can I function properly in this activity

**Reply by olegcherev** [reply]
Hi @damonexsppfx-alt — if you mean “how do I show up / use this properly,” here is the current behavior:

1. **Install** the GitHub App on the org/repos you care about.
2. Contributors are people with activity on **those installed repos** (commits, PRs, reviews, issues) — not every fork on GitHub.
3. Fork authors are **not** included automatically today. That is exactly this feature request: count fork contributors, at least on the Top Contributors widget.
4. Until that ships, the workaround is to **add the fork repo** (or the people you want counted) to the workspace, or keep installing on the upstream repo where the work actually lands.

If you tell us whether you are on the **upstream** repo or a **fork**, we can point you to the exact next click.

Related docs: https://devactivity.com/help/reports/xp-contribution-score-overview/
